pkg/alertmanager/README.md

```diff
@@ -0,0 +1,108 @@
+## Alertmanager Package
+
+The alertmanager package and CRDs together make up the alertmanager integration in the prometheus operator. Here is a
+quick overview of the overall package:
+
+![pkg/alertmanager and crds](../../Documentation/img/am_readme.png)
+
+### CRDs
+
+Prometheus operator supports two CRDs related to Alertmanager.
+
+- Alertmanager CRD: Defines the alertmanager statefulset.
+- AlertmanagerConfig CRD: Deines the config for the alertmanager instances.
+
+The Alertmanager CRD is in [v1](https://github.com/prometheus-operator/prometheus-operator/blob/main/README.md#project-status).
+while AlertmanagerConfig CRD is currently in [beta](https://github.com/prometheus-operator/prometheus-operator/blob/main/README.md#project-status).
+
+These CRDs are then picked up, validated and converted by the operator into configs.
+
+### Updating AlertmanagerConfig CRD
+
+If you are looking to make changes to the AlertmanagerConfig CRD, you would probably be making changes to the following files and for the reasons mentioned
+
+  <b>
+  pkg/apis/monitoring/v1alpa1/alertmanager_config_types.go 
+  </b>
+
+  <br>
```

Comment:
No need for `<br>`. Just add two spaces at the end of the line where you want a newline **mdox** generally adds an extra line break for `<br>` which can be avoided by this method.

Reply:
This might help https://github.github.com/gfm/#example-660